lib/pages/App.js

```diff
@@ -247,9 +247,12 @@ export default class App extends Component {
             createUser={this.props.async.createCustodialAccount}
             updateUser={this.props.async.clickEditUserNext}
             cancelEdit={_.partial(this.handleClickChangePerson, {metric: {eventName: metrics.CLINIC_ADD_CANCEL}})}
-            errorMessage={this.props.createCustodialAccountErrorMessage}
-            errorDismissed={this.props.createCustodialAccountErrorDismissed}
+            createErrorMessage={this.props.createCustodialAccountErrorMessage}
```

Comment:
`createErrorMessage` => `createCustodialAccountErrorMessage` which give a better indication of what we are creating and matches `updateProfileErrorMessage={this.props.updateProfileErrorMessage}`  better ...